src/components/Icon/MinusCircleIcon/MinusCircleIcon.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-MinusCircleIcon');
+
+/**
+ *
+ * {"categories": ["visual design", "icons"], "extend": "Icon", "madeFrom": ["Icon"]}
+ *
+ * Description.
+ */
+const MinusCircleIcon = createClass({
+	displayName: 'MinusCircleIcon',
+	propTypes: {
```

Comment:
like we discussed in slack, i think we should stop doing this. This component essentially doesn't have any props, everything is passed through to `Icon`, which has propType declarations, so I think we're just creating console noise with these redundant propType checks